curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -277,7 +279,18 @@ void FuseS3Client::WarmUpAllObjs(
                 delete []context->buf;
                 return;
             }
-            // todo: retry
+            if (context->retry >= downloadMaxRetryTimes) {
+                if (pendingReq.fetch_sub(1, std::memory_order_seq_cst) == 1) {
+                    VLOG(6) << "pendingReq is over";
+                    cond.Signal();
+                } else {
+                    LOG(WARNING) << "Up to max retry times";
```

Comment:
no need else？ because we always need this output.  and please add the objname that is download failed